controllers/manifests/base/pvc.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: {}
```

Comment:
Even though it's meant to be processed by kustomization.yaml, maybe we would want to keep valid kube yaml here, e.g. `storage: 10Gi``.

Unless it's preferred to keep it as-is so it doesn't get mistaken for one of the example files.